### PR TITLE
Xcode project setup

### PR DESCRIPTION
**Pull request by anastasiiaiurok-okta**
Re-organized project to a Xcode framework project.

### CODE REVIEW

File: Example/AppDelegate.swift [hunk 1]
```diff
@@ -31,10 +31,6 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     }
 
     func application(_ app: UIApplication, open url: URL, options: [UIApplication.OpenURLOptionsKey : Any]) -> Bool {
```
Comment (IldarAbdullin-okta) [inline Example/AppDelegate.swift]: Remove this function at all?

File: Okta/OktaOidc/OktaOidc.swift [hunk 2]
```diff
@@ -15,7 +15,7 @@ import UIKit
 public class OktaOidc: NSObject {
 
     // Current version of the SDK
-    @objc public static let VERSION = "3.1.0"
+    @objc public static let VERSION = "3.2.0"
```
Comment (IldarAbdullin-okta) [inline Okta/OktaOidc/OktaOidc.swift]: This is not required anymore, as framework version should be now in plist file. Please remove

Comment (anastasiiaiurok-okta) [reply Okta/OktaOidc/OktaOidc.swift]: Done.

File: Example/AppDelegate.swift (hunk 1 above)
Comment (anastasiiaiurok-okta) [reply Example/AppDelegate.swift]: Sorry, this was removed occasionally. We still need it for iOS < 11.

**Comment by IldarAbdullin-okta** [comment]
Please update readme file with carthage instructions

**Pull request merged**